global.json

```diff
@@ -9,10 +9,10 @@
     "vs": {
       "version": "17.6.0"
     },
-    "xcopy-msbuild": "17.6.0"
+    "xcopy-msbuild": "17.6.0-2"
```

Comment:
the roslyntools package has a -2 at the end, but vs on the machine needs 17.6.0

Reply:
> the roslyntools package has a -2 at the end

Any idea why?

Reply:
I think it might be for 17.6.2?  But not really sure.  The package is here - https://dnceng.visualstudio.com/public/_artifacts/feed/dotnet-eng/NuGet/RoslynTools.MSBuild/versions/17.6.0-2